### ISSUE TITLE
Is this cloud space removal

### ISSUE THREAD

**Issue by hahahahccccc** [leomuguchia/fmi_off#1]
Is this cloud space removal

**Comment by leomuguchia** [comment]
yes as long as the device isnt listed as stolen, it should work. check your device status first before trying it. if its marked as stolen unfortunately cant help you out on that. if its not and your unable reach me out and ill do it for you for a fee

**Comment by salim560** [comment]
Bro please me do fmi off for me I downloaded the scripts but I don't know to used it

**Comment by leomuguchia** [comment]
i do it for a price and as i have mentioned, it shouldnt be marked as lost
by owner

**Comment by salim560** [comment]
Bro please how much if I can pay

**Comment by salim560** [comment]
Drop your telegram username to chat you

**Comment by leomuguchia** [comment]
@LioM1000

**Comment by leomuguchia** [comment]
iphone x and lower 50$
iphone 11 &12 - 100$
iphone 13 and above - 150$
